docs/changes/2305.feature.rst

```diff
@@ -0,0 +1,2 @@
+Implements the Image Pixel-wise fit for Atmospheric Cherenkov Telescopes reconstruction method
```

Comment:
Is this really true?

Shouldn't this be more like a bugfix release like:

> Fix the implementation of ImPACT 

?

Reply:
I'd say "Large overhaul of the implementation of ImPACT", but you're right in principle

Reply:
Good point, should have used what @ParsonsRD put in his PR #2052 that I based this PR on, I updated with some of the notes from there. Left some stuff out because I think it is no longer relevant.